pubsub/cloud-client/iam.py

```diff
@@ -51,6 +51,8 @@ def get_subscription_policy(project, subscription_name):
     print("Policy for subscription {}:".format(subscription_path))
     for binding in policy.bindings:
         print("Role: {}, Members: {}".format(binding.role, binding.members))
+
+    client.close()
```

Comment:
Now that I think more about this, it's probably best to use the "with" syntax wherever possible. ie: every place where the subscriber client and the close call are in the same function block.

Reply:
Acknowledged.